### PR TITLE
Split replies and status updates

### LINKED ISSUE #55: Option to hide @replies by default
Default landing page lists messages which are a reply by default, this makes it a little hard to follow. It's be nice to have an option to hide the thread by default.

### PR DESCRIPTION
**Pull request by mapkyca**
Split status messages and replies, allows for filtering on both.

* Detects both inreplyto and @reply as a reply
* Introduces creatable ContentType field (default true), which determines whether a button will appear on the create bar

(closes #55)

### CODE REVIEW

**Comment by mapkyca** [comment]
Bumping this. 

FWIW, I've been running this on my own site for a while, really handy for me.

**Comment by mapkyca** [comment]
Bump

**Comment by mapkyca** [comment]
Bump

**Comment by mapkyca** [comment]
Comments - the things that go bump in the night.

**Comment by benwerd** [comment]
Bump - and boom.

**Comment by mapkyca** [comment]
Yay!
